addons/airway/functions/script_component.hpp

```diff
@@ -20,3 +20,7 @@
 #define VAR_IN_PAIN           "ACE_medical_inPain"
 #define VAR_TOURNIQUET        "ACE_medical_tourniquets"
 #define VAR_FRACTURES         "ACE_medical_fractures"
+
+//
+#define VAR_SPO2              QEGVAR(breathing,airwayStatus)
```

Comment:
i wonder if that should be moved to main addon 🤔

Reply:
Would that be in main/script_macros.hpp?

Reply:
hm. leave it for now, i will make separate PR in future to create one suitable for that